ramls/patron-notice-policy.json

```diff
@@ -344,6 +344,7 @@
                   "Recall request",
                   "Recall loanee",
                   "Hold request",
+                  "Request Expiration",
```

Comment:
Yes, it's case sensitive. It has been decided to capitalize the first letter of the second word, because it had been done similarly for `Hold Expiration` option in patron-notice-policy json schema (https://github.com/folio-org/mod-circulation-storage/blob/master/ramls/patron-notice-policy.json#L349).

Reply:
It seems to me, `Hold Expiration` is the outlier here and that it should be corrected and not to further pollute the enumeration with mixed case second words.

Reply:
@mreno-EBSCO in this case corresponding changes in the ui module must be done as ui can get an error trying to interact with `patron-notice-policy-storage` interface

Reply:
Sure, I realize that there would be changes required by clients of this API, but I believe striving for consistency is crucial to quality APIs. Maybe we need another JIRA to make the enumerations values consistent with respect to case... or we can just ignore this inconsistency. What do you think @marcjohnson-kint?

Reply:
Are there any changes required in the ui module for Request Expiration if we changed it to lower case "e"?

Reply:
No, it can be safely changed. Request expiration option is not presented on the ui yet.

Reply:
Can we change it to lower case "e" now that we know that upper casing the first letter of the second or subsequent words in the enum is not recommended?  We could deal with 'Hold Expiration' in another JIRA, as @mreno-EBSCO suggested.

Reply:
@mreno-EBSCO My understanding is that FOLIO decided to standardise on first word capitalization when defining states / statuses / enumerated values e.g. `Checked out`.

With the exception that the first word after hyphenation is also capitalized e.g. `Open - Awaiting pickup`.

I would prefer we be consistent if possible without incurring significant coordination overhead (e.g. if we need to break a big, heavily used interface).

It is decisions like this where I'm advocating for many interfaces to spend some time as experimental before 1.0.